src/data/products.js

```diff
@@ -1,30 +1,39 @@
 import hoodie from '../assets/images/products/hoodie.png';
 import headphone from '../assets/images/products/headphone.png';
 import longCoat from '../assets/images/products/long-coat.png';
+import getRandomArbitraryNumber from '../utility';
+import { SIZES } from '../utility/appConstants';
 
 const products = [
   {
     id: 1,
-    name: 'Classic Hoodie',
-    price: '599',
-    rating: 5,
+    title: 'Classic Hoodie',
+    price: 4097,
     image: hoodie,
     description:
-      '-UNISEX Funky PRINTED COTTON ,FULL SLEEVE Sweatshirt Hoodie & Kangaroo POCKETS and CAP with print"MARVEL". Same as shown in picture. -Material: 100% COTTON SWEATSHIRT -Best suitable for winter season, cool places, inner fleece material keep you WARM -High Quality PRINT which lays down soft, durable, full color spectrum prints that last forever -PLEASE REFER SIZECHART WHICH ATTACHED WITH MODEL PHOTOSHOOT.',
+      'great outerwear jackets for Spring/Autumn/Winter, suitable for many occasions, such as working, hiking, camping, mountain/rock climbing, cycling, traveling or other outdoors. Good gift choice for you or your family member. A warm hearted love to Father, husband or son in this thanksgiving or Christmas Day.',
+    rating: getRandomArbitraryNumber(1, 5),
+    sizes: SIZES,
   },
```

Comment:
## This is a proposed structure of each product item. 

Let's discuss this structure further if needed. Once we decide this, please update all the products, or just add fewer products for now with these details

```js
{
      title: 'Embroidered Semi Stitched Lehenga Choli  (Pink, Blue)',
      description: 'description'
      category: "Women's Lehenga Cholis",
      featured: false,
      fastDelivery: true,
      productId: 'B01N2K14U7',
      bestSeller: false,
      price: {
        before_price: 0,
        currency: 'INR',
        current_price: '1599',
        discounted: true,
        savings_amount: 1194,
        savings_percent: 74,
      },
      reviews: { rating: 3.7, total_reviews: 15209 },
      sizes: ['X', 'XS'],
      sponsored: false,
      thumbnail:
        'https://rukminim1.flixcart.com/image/781/937/jm573ww0/lehenga-choli/4/w/n/free-mdl19-define-jewellery-original-imaf948fythg85yp.jpeg?q=50',
      gallery: [
        'https://rukminim1.flixcart.com/image/781/937/jm573ww0/lehenga-choli/4/w/n/free-mdl19-define-jewellery-original-imaf948fythg85yp.jpeg?q=50',
        'https://rukminim1.flixcart.com/image/781/937/jm573ww0/lehenga-choli/4/w/n/free-mdl19-define-jewellery-original-imaf948fgv5jhpkg.jpeg?q=50',
      ],
      url: `${process.env.REACT_APP_FRONT_END_URL}/product-details/B01N2K14U7`,
    },
```



## I've gathered two product data, feel free to use them:

<details>
  <summary>
    Product details
  </summary>

```js

products: [
    {
      title: 'Embroidered Kanjivaram Georgette Saree  (Dark Blue)',
      description: 'description',
      sizes: ['XS', 'S', 'M'],
      category: 'Sarees',
      featured: false,
      fastDelivery: true,
      productId: 'B087623TMW',
      bestSeller: false,
      price: {
        before_price: 0,
        currency: 'INR',
        current_price: '629.00',
        discounted: true,
        savings_amount: 62.9,
        savings_percent: 10,
      },
      reviews: { rating: 4.6, total_reviews: 520 },
      sponsored: false,
      thumbnail:
        'https://rukminim1.flixcart.com/image/781/937/jz1l7rk0/sari/b/k/b/free-k-s-009-kapuriya-fab-original-imafdzu7ej6srht6.jpeg?q=50',
      gallery: [
        'https://rukminim1.flixcart.com/image/781/937/jz1l7rk0/sari/b/k/b/free-k-s-009-kapuriya-fab-original-imafdzu7ej6srht6.jpeg?q=50',
        'https://rukminim1.flixcart.com/image/781/937/k3vwxow0/sari/r/y/m/free-arvy-kuki-fashion-original-imafdzu8vh3dgvpn.jpeg?q=50',
      ],
      url: `${process.env.REACT_APP_FRONT_END_URL}/product-details/B087623TMW`,
    },
    {
      title: 'Embroidered Semi Stitched Lehenga Choli  (Pink, Blue)',
      description: 'description',
      sizes: ['X', 'XS'],
      category: "Women's Lehenga Cholis",
      featured: false,
      fastDelivery: true,
      productId: 'B01N2K14U7',
      bestSeller: false,
      price: {
        before_price: 0,
        currency: 'INR',
        current_price: '1599',
        discounted: true,
        savings_amount: 1194,
        savings_percent: 74,
      },
      reviews: { rating: 3.7, total_reviews: 15209 },
      sponsored: false,
      thumbnail:
        'https://rukminim1.flixcart.com/image/781/937/jm573ww0/lehenga-choli/4/w/n/free-mdl19-define-jewellery-original-imaf948fythg85yp.jpeg?q=50',
      gallery: [
        'https://rukminim1.flixcart.com/image/781/937/jm573ww0/lehenga-choli/4/w/n/free-mdl19-define-jewellery-original-imaf948fythg85yp.jpeg?q=50',
        'https://rukminim1.flixcart.com/image/781/937/jm573ww0/lehenga-choli/4/w/n/free-mdl19-define-jewellery-original-imaf948fgv5jhpkg.jpeg?q=50',
      ],
      url: `${process.env.REACT_APP_FRONT_END_URL}/product-details/B01N2K14U7`,
    },
  ],

```

<details>

Reply:
Good work there! I'll surely use this and the structure looks fine although it contains more than the required information, but we can always trim that.

Reply:
I thought about those details. For example, we could have a discounted price etc. 

Specifically mention if you wanna add or remove something.

Reply:
I'm keeping those details for now although removed a few:
- before_price
- savings_percent
because I don't think we need much detail here as we currently don't have a backend and when we would have a backend we'll have to also manage these calculations

Reply:
We need those to show the difference in offers. Once we have the backend, we'd make the API to return in this format.

Reply:
we don't have that functionality now, let's inculcate them later.